storey/table.py

```diff
@@ -685,12 +686,15 @@ def new_from_name(aggregation, max_value=None, set_data=None):
         elif aggregation == 'first':
             return FirstValue(max_value, set_data)
 
-    def _set_value(self, value):
-        if self._max_value and value > self._max_value:
+    def _set_value_with_max(self, value):
+        if value > self._max_value:
```

Comment:
is it really needed? 
you anyway have to check if max_value, why does it matter if it's done in _set_value or in the calling function?
it's just complicates the logic imo

Reply:
Because it happens once in init instead of 5M times.